repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/publicationprofessions.yaml

```diff
@@ -0,0 +1,60 @@
+/content/documents/administration/valuelists/publicationprofessions:
```

Comment:
Could we `snake_case` the ValueList keys [`selection:key`] just to be inline with other ValueLists ?